src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+}
+
+.search {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.searchSmall {
+  height: 42px;
+}
+
+.img {
+  width: 19px;
+}
+
+.imgSmall {
+  width: 11px;
+}
+
+.input {
+  box-sizing: border-box;
+  width: 100%;
+  outline: none;
+  padding: 0 0 0 62px;
+  font-family: inherit;
+  height: 70px;
+  font-size: 16px;
+  font-weight: 800;
+  line-height: 22px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.inputSmall {
+  padding: 0 0 0 33px;
+  height: 42px;
+  font-size: 14px;
+  font-weight: 900;
+  line-height: 19px;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.inputSmall::placeholder {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputSmall:focus {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.button {
+  border: 0;
+  cursor: pointer;
+  background-color: transparent;
+  position: absolute;
+  margin-top: 25px;
```

Comment:
You have given an absolute position to buttons, so it would be better to use top/left/right/bottom rules to center this button instead of margins